libraries/stdlib/js/src/js/rtti.js

```diff
@@ -39,22 +39,35 @@ Kotlin.callSetter = function (thisObject, klass, propertyName, value) {
     Kotlin.callSetter(thisObject, Object.getPrototypeOf(klass), propertyName, value);
 };
 
-function isInheritanceFromInterface(ctor, iface) {
-    if (ctor === iface) return true;
-
+function getAllInterfaces(ctor) {
+    if (ctor.$metadata$ == null) ctor.$metadata$ = {};
     var metadata = ctor.$metadata$;
-    if (metadata != null) {
+    if (metadata._allInterfaces === undefined) {
```

Comment:
would be better to use `void 0` instead of `undefined`

Reply:
Makes sense.